tests/conftest.py

```diff
@@ -49,40 +49,85 @@ def init_openshift_deployer():
     return Sandcastle(image_reference=NON_EX_IMAGE, k8s_namespace_name=PROJECT_NAME)
 
 
+@pytest.fixture(scope="session")
+def build_now():
+    """ build a container image with current sandcastle checkout """
+    project_root = Path(__file__).parent.parent
+    run_command(
+        [
+            "docker",
+            "build",
+            "-t",
+            TEST_IMAGE_NAME,
+            "-f",
+            "Dockerfile.tests",
+            str(project_root),
+        ]
+    )
+
+
 # TODO: refactor into a class
-def run_test_within_pod(test_path: str, with_pv_at: Optional[str] = None):
+def run_test_within_pod(
+    test_path: str, with_pv_at: Optional[str] = None, new_namespace: bool = False
+):
     """
     run selected test from within an openshift pod
 
     :param test_path: relative path to the test
     :param with_pv_at: path to PV within the pod
+    :param new_namespace: create new namespace and pass it via env var
     """
     config.load_kube_config()
     configuration = client.Configuration()
     assert configuration.api_key
     api = client.CoreV1Api(client.ApiClient(configuration))
 
-    container = {
+    pod_name = f"test-orchestrator-{get_timestamp_now()}"
+
+    cont_cmd = [
+        "bash",
+        "-c",
+        "ls -lha "
+        "&& id "
```

Comment:
the trailing whitespaces are intentional?

Reply:
yes, I don't know if it worked without them to be honest:
good: `ls -lha && id &&`
bad? `ls -lha&& id&&`